src/main/java/ch/njol/skript/expressions/ExprCoordinate.java

```diff
@@ -85,7 +95,11 @@ protected String getPropertyName() {
 	@Override
 	public void change(final Event e, final @Nullable Object[] delta, final ChangeMode mode) throws UnsupportedOperationException {
 		assert delta != null;
-		final Location l = getExpr().getSingle(e);
+		Object o = getExpr().getSingle(e);
+		if (o instanceof Chunk) {

```

Comment:
if chunks aren't possible to change, can we error in acceptsChange if the return type is known?